src/TRegx/DataProvider/CrossDataProviders.php

```diff
@@ -3,6 +3,11 @@
 
 class CrossDataProviders
 {
+    /**
+     * @param array<array-key, array<int, mixed>> ...$dataProviders
```

Comment:
I'm not quite sure whether the types for parameters with the splat operator are correct, though. Could you please double-check when reviewing? Thanks!

Reply:
What IDE are you using? Is it PhpStorm?

Reply:
I'm using PhpStorm, and for https://github.com/MyIntervals/emogrifier, we're using Psalm for static type analysis.

Reply:
(PHPStan also recognizes the array<int, foo> syntax.)

Reply:
And I'm using the "PHP Inspections (EA Ultimate)" (i.e., the paid version of the similarly-named free plugin) PhpStorm plugin for more powerful static analysis in PhpStorm.

Reply:
@oliverklee I think conforming to Psalm recommendations and syntax is the way to go. @iquito What so you think?